### PR TITLE
[SMALLFIX] Converted fields to local variables

### PR DESCRIPTION
**Pull request by HorizonNet**
Converted some fields to local variables in the *LineageFileSystemTest* class.

### CODE REVIEW

File: core/client/src/test/java/alluxio/client/lineage/LineageFileSystemTest.java [hunk 1]
```diff
@@ -55,8 +53,9 @@ public void before() {
     Whitebox.setInternalState(LineageContext.class, "INSTANCE", mLineageContext);
     mAlluxioLineageFileSystem = LineageFileSystem.get();
     Whitebox.setInternalState(mAlluxioLineageFileSystem, "mLineageContext", mLineageContext);
-    mFileSystemContext = PowerMockito.mock(FileSystemContext.class);
-    mFileSystemMasterClient = PowerMockito.mock(FileSystemMasterClient.class);
+    FileSystemContext mFileSystemContext = PowerMockito.mock(FileSystemContext.class);
```
Comment (gpang) [inline core/client/src/test/java/alluxio/client/lineage/LineageFileSystemTest.java]: local variables do not need to be prefixed by 'm'.

**Comment by gpang** [comment]
@HorizonNet Thanks for the fix! I left a minor comment.

**Comment by HorizonNet** [comment]
@gpang Thanks for the comment. I updated the PR accordingly.

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**